gpg/gpgtest/gpg.go

```diff
@@ -0,0 +1,77 @@
+package gpgtest
```

Comment:
I've been thinking for a while that we should have a package for testing machinery. Wouldn't it make sense to have a common place were to put all the stuff?

On the other hand having the test machinery closer to the code in which it's used probably makes it easier to understand.

Reply:
There are a few surprises, like test files way down in kubernetes/cluster/testfiles being used in several places. (Maybe the solution to that is not to use the same fixtures in several places ..)

Reply:
> On the other hand having the test machinery closer to the code in which it's used probably makes it easier to understand.

I agree.

> (Maybe the solution to that is not to use the same fixtures in several places ..)

This could be a solution although I see downsides to this too (my experience with almost identical fixtures in multiple places is that it results in high maintenance, new issues and/or frustrations).

---

Given the above, shall we:

a) move this to a new issue so we can fix it throughout the whole codebase
b) create our own fixtures in `gpg/gpgtest` so we have it fixed there
c) ...

Reply:
I would go for a)

Reply:
Created an issue for it.